internal/kubernetes/networkHandler.go

```diff
@@ -87,6 +89,44 @@ func (n *networkHandler) OnHandshakeSuccess(username string, clientVersion strin
 		if n.pod, err = n.cli.createPod(ctx, n.labels, n.annotations, env, nil, nil); err != nil {
 			return nil, err
 		}
+		if b64creds, ok := metadata["KRB5CCDATA"]; ok {
+			_, ok := metadata["KRB5CCPATH"]
+			if !ok {
+				return nil, fmt.Errorf("Provided credentials with no path")
+			}
+
+			credCtx, cancelFunc := context.WithTimeout(
+				context.Background(),
+				n.config.Timeouts.CommandStart,
+			)
+			defer cancelFunc()
+
+			exec, err := n.pod.createExec(credCtx, []string{n.config.Pod.AgentPath, "write-file", "/tmp/krb5cc"}, map[string]string{}, false)
```

Comment:
This does not seem to handle the `session` case, only the `connection` mode. This will cause problems for anyone running in `session` mode.